packages/protocol/contracts/common/AddressResolver.sol

```diff
@@ -132,7 +132,7 @@ abstract contract AddressResolver {
     }
 
     /**
-     * @notice Fetch the AddressManager's address.
+     * Fetch the AddressManager's address.
```

Comment:
Shall we force a rule for verb so here we use "Fetches" instead of "Fetch"?

Reply:
in the future yes. unless you are aware of a out of the box solution that we can quickly use for this now. otherwise we will integrate an AI based solution or use vale (https://github.com/errata-ai/vale-action) in our CICD after cleaning up protocol docs.

Reply:
actually fetch is a verb though lol

Reply:
actually i agree with you. i resorted to google's javadoc writing style guide. it is preferred to use a third person singular verb. i updated this.